marketplaceItems/appreciate/appResources/appData/appreciate_app.js

```diff
@@ -0,0 +1,947 @@
+            } else {
+                var props = INTENSITY_ENTITY_PROPERTIES;
+                props.position = halfwayBetweenHands();
+
+                currentInitialAngularVelocity.x = Math.random() *
+                    (INTENSITY_ENTITY_MAX_ANGULAR_VELOCITY.x -
+                    INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.x) + INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.x;
+                currentInitialAngularVelocity.y = Math.random() *
+                    (INTENSITY_ENTITY_MAX_ANGULAR_VELOCITY.y -
+                    INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.y) + INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.y;
+                currentInitialAngularVelocity.z = Math.random() *
+                    (INTENSITY_ENTITY_MAX_ANGULAR_VELOCITY.z -
+                    INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.z) + INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.z;
+                props.angularVelocity = currentInitialAngularVelocity;
+
+                intensityEntity = Entities.addEntity(props, "avatar");
+                
+                var materialProps = INTENSITY_MATERIAL_ENTITY_PROPS;
+                materialProps.parentID = intensityEntity;
+                intensityMaterialEntity = Entities.addEntity(materialProps, "avatar");
+            }
+        } else {
+            if (intensityEntity) {
+                Entities.deleteEntity(intensityEntity);
+                intensityEntity = false;
+            }
+            if (intensityMaterialEntity) {
+                Entities.deleteEntity(intensityMaterialEntity);
+                intensityMaterialEntity = false;
+            }
+            
+            maybeClearUpdateIntensityEntityInterval();
+        }
+    }
+
+    // Function that AppUI calls when the App's UI opens
+    function onOpened() {
+        updateCurrentIntensityUI();
+    }
+
+    // Locally pre-caches all of the sounds in the sounds/claps and sounds/whistles
+    // directories.
+    var NUM_CLAP_SOUNDS = 16;
+    var NUM_WHISTLE_SOUNDS = 17;
+    var clapSounds = [];
+    var whistleSounds = [];
+    function getSounds() {
+        for (var i = 1; i < NUM_CLAP_SOUNDS + 1; i++) {
+            clapSounds.push(SoundCache.getSound(Script.resolvePath(
+                "resources/sounds/claps/" + ("0" + i).slice(-2) + ".wav")));
+        }
+        for (i = 1; i < NUM_WHISTLE_SOUNDS + 1; i++) {
+            whistleSounds.push(SoundCache.getSound(Script.resolvePath(
+                "resources/sounds/whistles/" + ("0" + i).slice(-2) + ".wav")));
+        }
+    }
+
+    // Locally pre-caches the Cheering and Clapping animations
+    var whistlingAnimation = false;
+    var clappingAnimation = false;
+    function getAnimations() {
+        var animationURL = Script.resolvePath("resources/animations/Cheering.fbx");
+        var resource = AnimationCache.prefetch(animationURL);
+        var animation = AnimationCache.getAnimation(animationURL);
+        whistlingAnimation = { url: animationURL, animation: animation, resource: resource};
```

Comment:
fixing the spaces inside the object

Reply:
by the brackets.  either both with a space or both without.  Not sure what the coding standards are for that one.

Reply:
Maybe consider prefixing the above 3 with whistling and the bottom 3 a new set of variables with clapping.

Reply:
ah yes, ty for catching spacing (this is copied/pasted from somewhere else in our codebase)